css/style.css

```diff
@@ -327,7 +327,7 @@ nav li {
   position: absolute;
   top: 40px;
   width: 305px;
-  height: 220px;
+  height: 250px; /*220px;*/
```

Comment:
Please don't keep old values commented out. Either change it for a reason or don't touch it. Otherwise the code turns slowly in a graveyard of old-lost-code.